.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,6 @@
+blank_issues_enabled: true
+
+contact_links:
+  - name: Ask a question
+    url: https://github.com/privateerproj/privateer/discussions/new
```

Comment:
Would you be opposed to us disabling discussions and then we can point people to open a new issue for questions?

It's too rare for people to actually use discussions correctly, and those end up being harder to follow because of it.

Reply:
Not opposed at all. Pushing change in both repos shortly.